### ISSUE TITLE
Backup sollet.io wallet

### ISSUE THREAD

**Issue by gulyasbence** [project-serum/spl-token-wallet#61]
Is it possible to export the private key of the wallet or to reveal the seed phrase?

**Comment by armaniferrante** [comment]
Yes it is possible. You can export your private key for a given account via the `Export` button on the bottom right of your SOL account balance card.

![Screenshot from 2020-12-12 10-35-07](https://user-images.githubusercontent.com/6990215/101992116-bc183a80-3c65-11eb-9bf8-4f51aaafb77f.png)

**Comment by gulyasbence** [comment]
Thanks for the answer @armaniferrante! Can I restore my existing wallet with this private key? When I select the restore existing wallet option I only have an option to enter the seed words, but I don't have that anymore.

**Comment by armaniferrante** [comment]
@gulyasbence all wallets need to have a Mnemonic key. In addition to the mnemonic's accounts, you can import individual accounts with your private key directly, using the UI displayed above.
